lib/cmd/fh2/init.js

```diff
@@ -28,8 +28,9 @@ module.exports = function init(argv, callback) {
   // We use stream type console output to prevent automatic newline.
   process.stdout.write(i18n._("Fetching project ZIP from GitHub... "));
 
+  var proxyRequest = request.defaults({'proxy': fhc.config.get("proxy")})
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)